python/connectors/connectors_auto_start.py

```diff
@@ -0,0 +1,121 @@
+# Checks if connector has run in the last 24 hours.  If not, launches it.
+
+import os
+import sys
+import json
+import requests
+from datetime import datetime, timedelta
+from prettytable import PrettyTable
+
+# Returns connector information from the List Connectors API.
+def get_connectors(base_url, headers):
+    connectors = []
+    list_connectors_url = base_url + "connectors"
+
+    response = requests.get(list_connectors_url, headers=headers)
+    if response.status_code != 200:
+        print(f"List Connector Error: {response.status_code} with {list_connectors_url}")
+        sys.exit(1)
+    
+    resp_json = response.json()
+    connectors = resp_json['connectors']
+
+    return connectors
+
+# Gets the connector runs for a the specified connector ID.
+def get_connector_runs(base_url, headers, connector_id):
+    connector_runs = []
```

Comment:
I think this is a copy paste thing. This variable does not appear to be used.

Reply:
Removed.  The function started out to do something a little different and this variable was left behind.